links.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft/hyper-schema#",
```

Comment:
When the draft 4 version had the URI of draft-4/hyper-schema, why do we not use the URI of draft-5/hyper-schema? (Or will that be a later PR?)